### PR TITLE
Admin AGL list for media

### PR DESCRIPTION
**Pull request by Floby**
some notable added features
- one click selection when selecting from popup (tinymce)
- indicate how many times the media is referenced by other contents
- group media by sha1
  - disable editing, deleting and whatnot for grouped media
  - display link to another admin page listing only media with this sha1
  - offer button back from that list to the main media list

there are some TODOs left
- move some use detection back to Centurion_Db_Table_[Row_]Abstract because it is generic enough to be widely used.
- admin list looks dead ugly when dealing with anything else than images
- generic form elements allowing to pick directly from the mediacenter (not just from rte fields)

### CODE REVIEW

File: library/Centurion/Contrib/media/views/scripts/admin-media/media.json.phtml [hunk 1]
```diff
@@ -15,16 +16,39 @@ foreach ($this->result as $key => $result) {
         $cover = $this->row;
 
     if (isset($this->popup)) :
-        $editUrl = 'javascript:mySubmit(\'' . $cover->getStaticUrl() . '\', \''.$this->escape($cover->filename).'\', \'\')';
+        $editUrl = 'javascript:mySubmit(\'' . $this->GetCdnUrl($cover->getStaticUrl()) . '\', \''.$this->escape($cover->filename).'\', \'\')';
```
Comment (lchenay) [inline library/Centurion/Contrib/media/views/scripts/admin-media/media.json.phtml]: GetCdnUrl ?

Comment (Floby) [inline library/Centurion/Contrib/media/views/scripts/admin-media/media.json.phtml]: that's specific to my project, i'm gonna edit that

**Comment by Floby** [comment]
@lchenay Removed trace of getCdnUrl and view var displayDuplicate is always boolean

**Comment by lchenay** [comment]
Can't be merge. I have a test failed:
- Centurion_Form_ModelTest::testSaveManyToMany()

2 unknown variables in Centurion_Form_Model_Abstract:822
- $refLocalArray
- $refForeignArray

See my branch merge/floby/feature-mediacenter

**Comment by Floby** [comment]
I merged with your merge/floby/feature-mediacenter and reverted the faulty commit.
apparently the Form_Model_Abstract doesn't use references on interesection tables for many to many relationships.

**Pull request merged**